.jest/setup/jestDomExpects.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-internal-modules
```

Comment:
`import/no-extraneous-dependencies` comes from importing a dev dependency? I see it's repeated quite often. Can we add it to override (for `.jest` and `.storybook`) in `.eslintrc`?